src/Features/VisualBasic/Portable/NavigationBar/VisualBasicNavigationBarItemService.vb

```diff
@@ -361,14 +361,12 @@ Namespace Microsoft.CodeAnalysis.NavigationBar
                     eventContainer.Name,
                     eventContainer.GetGlyph(),
                     indent:=1,
-                    spans:=allMethodSpans.ToImmutableArray(),
                     childItems:=rightHandMemberItems.ToImmutableArray())
             Else
                 Return New ActionlessItem(
                     String.Format(VBFeaturesResources._0_Events, containingType.Name),
                     Glyph.EventPublic,
                     indent:=1,
-                    spans:=allMethodSpans.ToImmutableArray(),
                     childItems:=rightHandMemberItems.ToImmutableArray())
```

Comment:
so this loks like we're losing data.  However here's why this is safe:

1. Only VB makes these ActionlessItem's
2. We literally do *nothing* when an ActionlessItem is clicked on.  You can see that here: https://github.com/dotnet/roslyn/blob/e126f50fec72721c05eaf2cc51c4163424be85ef/src/EditorFeatures/VisualBasic/NavigationBar/VisualBasicEditorNavigationBarItemService.vb#L76-L85

As we can, VB *only* supports doing something when a SymbolItem or AbstractGenerateCodeItem was the one selected.  ActionlessItems are solely for having a hierarchy in the UI.